### PR TITLE
use SASS instead of precss

### PR DESCRIPTION
**Pull request by just-boris**
There is no future in precss, also it makes use some problems in development, like bad support in Idea.

Sass would be better there.

### CODE REVIEW

File: allure-generator/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ apply plugin: 'maven'
 apply plugin: 'com.moowork.node'
 
 node {
-    version = '7.10.0'
+    version = '8.1.3'
```
Comment (just-boris) [inline allure-generator/build.gradle]: Not necessary for this particular feature, but good to have

File: allure-generator/webpack.config.js [hunk 2]
```diff
@@ -35,17 +33,26 @@ module.exports = (env) => {
...
                         fallback: 'style-loader',
                         use: [
                             'css-loader',
-                            postcssLoader
+                            postcssLoader,
+                            'sass-loader'
                         ]
                     })
             }, {
+                test: /\.css$/,
```
Comment (just-boris) [inline allure-generator/webpack.config.js]: we still support plain css files, that are imported from libraries (e.g. font-awesome)

File: allure-generator/webpack.config.js [hunk 3]
```diff
@@ -7,8 +7,6 @@ const postcssLoader = {
     loader: 'postcss-loader',
     options: {
         plugins: [
-            require('postcss-import'),
-            require('precss')({'import': {disable: true}}),
             require('autoprefixer')
```
Comment (just-boris) [inline allure-generator/webpack.config.js]: PostCSS is still used, as we want to have autoprefixer. (To have less pain supporting IE and Safari)

**Comment by baev** [comment]
@just-boris thanks!

**Pull request merged**